test-cases/longevity/longevity-ics-toggle-strategy-large-partitions-24h.yaml

```diff
@@ -0,0 +1,51 @@
+test_duration: 1440
```

Comment:
Need to wait for the official new longevity from @juliayakovlev for large partitions.
Then, should use a second test-config file that just override some parameters like duration and nemesis.

Reply:
Done. @roydahan what different duration does it need?